virttest/test_setup.py

```diff
@@ -1043,13 +1043,18 @@ def _release_dev(self, pci_id):
             logging.info("Run command in host: %s" % cmd)
             try:
                 output = None
-                output = process.system_output(cmd, timeout=60)
+                output = process.system_output(cmd, shell=True, timeout=60)
             except Exception:
                 msg = "Command %s fail with output %s" % (cmd, output)
                 logging.error(msg)
                 return False
         if self.is_binded_to_stub(pci_id):
             return False
+        else:
+            for pf in self.pf_vf_info:
+                for vf in pf["vf_ids"]:
+                    if vf["vf_id"] == pci_id:
+                        vf["occupied"] = False
```

Comment:
Yes I have tested it. After my changes I see the VFs getting back to free pool [occupied=False] after the specific operation [say bind/unbind].